se2_planning/include/se2_planning/GridMapLazyStateValidator.hpp

```diff
@@ -15,6 +15,8 @@
 
 namespace se2_planning {
 
+enum StateValidityCheckingMethod : int { COLLISION = 0, TRAVERSABILITY = 1, ROBUST_TRAVERSABILITY = 2 };
```

Comment:
ah okay, I see it now. But we need to explain a bit what is behind these methods.

Reply:
Yes, I have just left all methods there that we can discuss/check what is actually useful. Improving the documentation is important for that, sounds good!